README.md

```diff
@@ -5,33 +5,43 @@ The Mozilla Lightbeam extension is a key tool for Mozilla to educate the public
 
 ## Quick Start
 
-* Clone the repository
-`git clone https://github.com/electrolyfish/lightbeam-we.git`
+### Clone the repository
 
-* There are a couple ways to try out this web extension:
-  1. Open Firefox and load `about:debugging` in the URL bar.
+**Note** This repository uses a [submodule](https://github.com/mozilla-services/shavar-prod-lists) to whitelist some third party requests. To ensure the submodule is cloned along with this repository, use a modified `clone` command:
+`git clone --recursive https://github.com/electrolyfish/lightbeam-we.git`
+
+### Run the web extension
+
+There are a couple ways to try out this web extension:
+
+1. Open Firefox and load `about:debugging` in the URL bar.
     - Click the [Load Temporary Add-on](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Temporary_Installation_in_Firefox) button and select the `manifest.json` file within the directory of this repository.
     - You should now see the Lightbeam icon on the top right bar of the browser.
     - Click the Lightbeam icon to launch the web extension.
     - Here is the screenshot:
 
     ![lightbeam-screenshot](/docs/images/lightbeam-launch.gif)
 
-  2. Install the [web-ext](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext) tool, change into the directory of this repository, and type `web-ext run`.
+2. Install the [web-ext](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext) tool, change into the directory of this repository, and type `web-ext run`.
     - This will launch Firefox and install the extension automatically.
     - This tool gives you some additional development features such as [automatic reloading](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext#Automatic_extension_reloading).
 
 ## Development Guide
 
-* Run `npm install`.
-* Run `npm run test` to check that everything is OK.
-  * If you have installed `eslint` globally, you will have to install globally the following `eslint` plugins too:
+### Download dependencies
+Run `npm install`.
+
+### Update the submodule
+To manually update the submodule at any time during development, run `git submodule update`.
```

Comment:
New users would use the modified `git clone --recursive https://github.com/electrolyfish/lightbeam-we.git` which would take care of init (referring to this [GitHub guide on submodules](https://github.com/blog/2104-working-with-submodules). They would only need an `init` command if they did not do the recursive clone command from the README.